src/main/kotlin/org/jitsi/rtp/extensions/Byte.kt

```diff
@@ -38,6 +38,19 @@ fun putBit(b: Byte, bitPos: Int, isSet: Boolean): Byte {
     }
 }
 
+/**
+ * Set or unset the bit or bits specified by [mask] of byte [b] according to [isSet]
+ * Unfortunately we can't do this as an extension function because
```

Comment:
This function doesn't modify `b`, it just reads it and produces a new value. Why can't it do the same as an extension function?

Reply:
I dunno, I was just copying the doc for the existing method right above it.

Reply:
I added this comment to the other method because my first thought, when implementing it, was to be able to do something like `myByte.setBit(2)`, and have that work but it can't: we can't modify `myByte` in-place.  I worried if it looked the same but relied on the caller using the return, it'd be easy to forget so I made it a separate function.  If we did make it an extension, I think we should be careful with the name of it so as to suggest it returns the modified value as opposed to modifying it in place.  I thought I remembered reading about a language feature to enforce using the return value (as opposed to ignoring it), but couldn't find anything just now.